rhine/src/FRP/Rhine/Clock/Realtime/Never.hs

```diff
@@ -0,0 +1,36 @@
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE TypeFamilies #-}
+
+-- | A clock that never ticks.
+module FRP.Rhine.Clock.Realtime.Never where
+
+-- base
+import Control.Concurrent (threadDelay)
+import Control.Monad (forever)
+import Data.Kind (Type)
+
+-- time
+import Data.Time.Clock
+
+-- rhine
+import FRP.Rhine.Clock
+import FRP.Rhine.Clock.Proxy
+
+-- transformers
+import Control.Monad.IO.Class
+
+-- | A clock that never ticks.
+data Never (tag :: Type) = Never
+
+instance (MonadIO m) => Clock m (Never tag) where
+  type Time (Never _) = UTCTime
+  type Tag (Never tag) = tag
```

Comment:
```suggestion
data Never = Never

instance (MonadIO m) => Clock m (Never tag) where
  type Time Never = UTCTime
  type Tag Never = Void
```

I believe it should be possible not having to specify the tag at all. By using `Void`, it really proves that no tag is ever produced. This has the advantage that if e.g. you have a `ParallelClock Never cl`, then `Tag (ParallelClock Never cl) = Either (Tag Never) (Tag cl)`, so when you encounter a `Left tag` value, you can always prove its absence using `absurd`.

Reply:
Nice idea. In places where we `initClock` we can simply map the type with either `undefined` or `absurd`.